string name
float32 confidence
uint32 bbox_x
uint32 bbox_y
uint32 width
uint32 height